scribus/plugins/import/pdf/slaoutput.cpp

```diff
@@ -3456,7 +3481,12 @@ void SlaOutputDev::endTextObject(GfxState *state)
 			else
 				ite = gElements.Items.first();
 			ite->setGroupClipping(false);
-			ite->setFillTransparency(1.0 - state->getFillOpacity());
+			if (importTextAsVectors == false) {

```

Comment:
a: if possible, always test for true : - )
b: the two branches seem to do the same

Reply:
ok, I'll flip the code over so it tests for true